### PR TITLE
update docs labels

### LINKED ISSUE #34115: Update docs to use recommended labels
**This is a Feature Request**
Requesting that docs be updated to use the Kubernetes [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) for service objects

**What would you like to be added**

Change app: MyApp examples to app.kubernetes.io/name: MyApp

**Why is this needed**

This caused some confusion for our engineers of what the best labels are to use and these docs examples often get copy pasted, having them be aligned to recommended labels makes sense.

**Comments**

N/A

### PR DESCRIPTION
**Pull request by TheRealDwright**
Updates docs to use kubernetes recommended labels
as outlined in the documentation.

- [en] update en docs to use recommended labels

Closes #34115

### CODE REVIEW

**Comment by bradtopol** [comment]
Assigning @kbarnard10 since this affects so much blog content
/assign @kbarnard10

**Review by joshdk — APPROVED** [review]
This looks great!

**Comment by Sea-n** [comment]
Hey @TheRealDwright,

Please limit this PR to only change `content/en`, and open different PR for each language (or leave it to other contributor).

Feel free to `/unhold` yourself once it's done.
/hold

**Comment by Sea-n** [comment]
Thanks! Then I can help with adjusting label.

/remove-language es fr id ja ko pt uk zh

File: content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md
Comment (sftim) [inline content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md:189]: We avoid changing blog articles that are more than 1 year old.

File: content/en/blog/_posts/2019-02-12-building-a-kubernetes-edge-control-plane-for-envoy-v2.md
Comment (sftim) [inline content/en/blog/_posts/2019-02-12-building-a-kubernetes-edge-control-plane-for-envoy-v2.md:41]: We avoid changing blog articles that are more than 1 year old.

**Review by sftim — CHANGES_REQUESTED** [review]
I'd prefer _not_ to revise the old blog articles. The other changes seem OK to me (I haven't checked thoroughly).

File: content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md
Comment (TheRealDwright) [reply content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md]: Please note the consistency in this file is an issue, L189 `app: MyApp` vs L229 `app: myapp`

Have removed the changes regardless.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:102]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:125]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:158]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:184]: This looks like a manual merge gone wrong; please fix it.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/dual-stack.md:311]: This looks like a manual merge gone wrong; please fix it.

**Review by sftim — CHANGES_REQUESTED** [review]
/remove-area blog

There are some merge conflicts that needed resolving manually but haven't been. @TheRealDwright, please fix those.

/hold
OK to unhold once the merge conflict byproducts are gone.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

File: content/en/docs/concepts/services-networking/dual-stack.md
Comment (TheRealDwright) [reply content/en/docs/concepts/services-networking/dual-stack.md]: Apologies, I must have missed this during one of the rebases I had to do.

**Comment by tengqm** [comment]
/hold cancel
Conflicts resolved.

**Comment by TheRealDwright** [comment]
@sftim is this something that there is any interest in merging?

**Review by Sea-n — COMMENTED** [review]
Thanks for your patient. The changes looks good to me now.
/lgtm

**Pull request merged**
